lib/resources/oidc.js

```diff
@@ -102,15 +103,22 @@ module.exports = (service, endpoint) => {
       const client = await getClient();
       const code_verifier = generators.codeVerifier(); // eslint-disable-line camelcase
 
+      // State is not strictly required because PKCS (code_challenge) protects us from CSRF attacks.
+      // See https://datatracker.ietf.org/doc/html/draft-ietf-oauth-security-topics#name-countermeasures-4
+      // We create a nonce (state) anyway because some IDPs (e.g. Authentik) will send the nonce
```

Comment:
`nonce` is the name of a different value which can be used at various points in OAuth/OIDC.  I think avoiding that word here would reduce chance of confusion.

Reply:
@openbrian just in case you missed this :slightly_smiling_face: